element_calcium_imaging/scan.py

```diff
@@ -432,6 +441,44 @@ def estimate_nd2_scan_duration(nd2_scan_obj):
                         for plane_idx in range(nd2_file.sizes.get("Z", 1))
                     ]
                 )
+        elif acq_software == "PrairieView":
+            import prairieviewreader
```

Comment:
I am out of the loop.  Is this a standard python package?  I don't see it on PyPI or GitHub.

Reply:
I'm wondering the same thing, @kushalbakshi can you clarify

Reply:
Ah I now see it here: https://github.com/kushalbakshi/prairieview-reader

Reply:
Perhaps we should add this function to [element-interface](https://github.com/datajoint/element-interface) or `element-calcium-imaging`.

Reply:
Yeah I can add it to `element-interface` and make a PR there if you prefer

Reply:
That would be great.  Thanks!